pkg/crc/machine/start.go

```diff
@@ -104,7 +112,10 @@ func (client *client) Start(ctx context.Context, startConfig StartConfig) (*Star
 	if err != nil {
 		return nil, errors.Wrap(err, "Cannot determine if VM exists")
 	}
+
 	if !exists {
+		telemetry.SetContextProperty(ctx, telemetryStartType, firstStart)
+
```

Comment:
Wondering if `firstStart` should be set in `startResult`, and then added to telemetry in the `cmd/` package? This would keep the `machine` package separate from `telemetry`/`context`. If returned as part of `startResult`, rather than `firstStart` this could be returned as `vmCreated` or similar name.

Reply:
startResult is not returned in case of error. It's good to know an error occurred in the context of a first start or so. 
I hope context will be also used by the daemon for cancellation. If the client aborts, then the daemon cut somewhere in a wait loop maybe.

Reply:
> startResult is not returned in case of error. It's good to know an error occurred in the context of a first start or so.

Indeed, but this does not mean the only way to achieve that is by adding a telemetry dependency to the machine package :) I also don't really consider "this package will be using cancellation through `context`" as a valid reason for making the package dependent on `telemetry`. I'm still undecided whether the `telemetry` package is a low-level one which is fine to use in most packages, or if we should try to keep it as a high level thing. 

Thinking while writing this message, I'd say what is missing is a slightly higher level telemetry API for use in other packages, ie some `telemetry.SetStartType()`, `telemetry.SetDiskSize()`, ... so that it's harder to misuse, and we keep some central list of telemetry data which is sent.

Reply:
I will do that!